app/bundles/CoreBundle/Helper/Dsn/Dsn.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
```

Comment:
Please declare strict_types in all new classes.
```suggestion

declare(strict_types=1);

```